_releases/0.17.0.1.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Bitcoin Core 0.17.0.1
+id: en-release-0.17.0.1
+name: release-0.17.0.1
+permalink: /en/releases/0.17.0.1/
+excerpt: Bitcoin Core version 0.17.0.1 is now available
+date: 2018-10-27
+
+## Use a YAML array for the version number to allow other parts of the
+## site to correctly sort in "natural sort of version numbers"
+release: [0, 17, 0, 1]
+
+## Optional magnet link.  To get it, open the torrent in a good BitTorrent client
+## and View Details, or install the transmission-cli Debian/Ubuntu package
+## and run: transmission-show -m <torrent file>
+#
+## Link should be enclosed in quotes and start with: "magnet:?
+#optional_magnetlink: "magnet:?xt=urn:btih:1c72f17bc1667a2ce81860b75135e491f6637d05&dn=bitcoin-core-0.17.0&tr=udp%3A%2F%2Ftracker.openbittorrent.com%3A80&tr=udp%3A%2F%2Ftracker.opentrackr.org%3A1337&tr=udp%3A%2F%2Ftracker.coppersurfer.tk%3A6969&tr=udp%3A%2F%2Ftracker.leechers-paradise.org%3A6969&tr=udp%3A%2F%2Fzer0day.ch%3A1337&tr=udp%3A%2F%2Fexplodie.org%3A6969"
+
+# Note: it is recommended to check all links to ensure they use
+#       absolute urls (https://github.com/bitcoin/bitcoin/doc/foo)
+#       rather than relative urls (/bitcoin/bitcoin/doc/foo).
+
+## Notes from bitcoin/bitcoin 2018-10-26 commit 1476554d3b1db5352c0de1de7524864d4bce3aac
+---
+{% include download.html %}
+{% githubify https://github.com/bitcoin/bitcoin %}
+Bitcoin Core version 0.17.0.1 is now available from:
+
+  <https://bitcoincore.org/bin/bitcoin-core-0.17.0.1/>
+
+This is a new major version release, including new features, various bugfixes
```

Comment:
Should this say "This is a minor bug fix for 0.17.0." (compare https://bitcoincore.org/en/releases/0.15.0.1/)

Reply:
Fixed upstream and here, thanks!